.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: python
 python:
   - "2.7"
+cache: pip
+
 install:
-  - 'pip install websocket-client pytest'
+  - pip install flake8 pytest websocket-client
+
+before_script:
+  # stop the build if there are Python syntax errors or undefined names
+  - flake8 .
+  # exit-zero treats all errors as warnings.
+  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=120 select=E,F,W --statistics
```

Comment:
You need to select `C90` as well to include the complexity check, and you're missing `--`, so: `--select=C90,E,F,W`.

You can specify `max-complexity` and `max-line-length` in the config file.

You can drop `--count` and `--statistics` since those are specified in the config.